Wikipedia/Code/AppDelegate.m

```diff
@@ -67,6 +67,7 @@ - (void)updateDynamicIconShortcutItems {
 #pragma mark - UIApplicationDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [UserHistoryFunnel.shared logSnapshot];
 #if WMF_IS_NEW_EVENT_LOGGING_ENABLED
     [[WMFEventLoggingService sharedInstance] start];
 #endif
```

Comment:
It might make sense to move everything above this line into `WMFAppViewController` after all the migrations have run. In general, it's nice to not load up `didFinishLaunchingWithOptions:` - I think Andy put it here initially while testing and we never moved it where it should be

Reply:
do you mean everything below line `70` should go into `WMFAppViewController`'s `launchAppInWindow`, after `[self migrateIfNecessary]`? Or do you mean just starting event logging shouldn't be here?

Reply:
Just ```[UserHistoryFunnel.shared logSnapshot];
#if WMF_IS_NEW_EVENT_LOGGING_ENABLED
    [[WMFEventLoggingService sharedInstance] start];
#endif```

Reply:
and can be added below `[self checkRemoteAppConfigIfNecessary];`

Reply:
err - actually scratch this for now, it can be a follow on if we get to it